lib/utils/ui/image_slider/simple_slider.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_poc/utils/ui/image_slider/dot_indicator.dart';
+
+class ImageSliderWidget extends StatefulWidget {
+  final List<String> imageUrls;
+  final BorderRadius imageBorderRadius;
+  final double imageHeight;
+
+  const ImageSliderWidget({
+    Key? key,
+    required this.imageUrls,
+    required this.imageBorderRadius,
+    this.imageHeight = 200.0,
+  }) : super(key: key);
+
+  @override
+  ImageSliderWidgetState createState() {
+    return ImageSliderWidgetState();
+  }
+}
+
+class ImageSliderWidgetState extends State<ImageSliderWidget>
+    with SingleTickerProviderStateMixin {
+  List<Widget> _pages = [];
+
+  final _pageController = PageController(initialPage: 0, keepPage: true);
+
+  @override
+  void initState() {
+    super.initState();
+    _pages = widget.imageUrls.map((url) {
+      return _buildImagePageItem(url);
+    }).toList();
```

Comment:
Please use the GetXController and put all the variables there instead in UI layer